src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagOptions.cs

```diff
@@ -28,6 +28,12 @@ public class FeatureFlagOptions
         /// </summary>
         public TimeSpan CacheExpirationInterval { get; set; } = RefreshConstants.DefaultFeatureFlagsCacheExpirationInterval;
 
+        /// <summary>
+        /// Forces feature flags and dynamic features to be written into the configuration system using the  
+        /// feature management V2.0 schema introduced in Microsoft.FeatureManagement 3.0.0-preview library.
+        /// </summary>
```

Comment:
I had a couple of new thoughts on this one.

* The field exists on FeatureFlagOptions, so it's already somewhat expected that the field is related to feature flags + feature management. 
* Specifying V2 in the name *may* get a little confusing since v3.0.0-preview of the feature management is required to consume it.

Do you think with those two things it's fine to just use `bool FeatureFlagOptions.EnablePreviewSchema` ?

Reply:
Do we want to keep symmetry with the option name we'll release with the stable version for opting out of V2 schema? 
Something like `bool FeatureFlagOptions.UseV1Schema` ? If we need symmetry, how about calling this preview option `bool FeatureFlagOptions.UseV2SchemaPreview`?

Reply:
> Do we want to keep symmetry with the option name we'll release with the stable version for opting out of V2 schema

I don't consider it a requirement. 

If we did though, we'd need to settle on what we want that to be. I don't feel good about the idea of having version as part of a field/method/type name.

So I imagine the opt-out api usage would be something like `featureFlagOptions.Schema = FeatureFlagSchema.V1`